scripts/gha/print_matrix_configuration.py

```diff
@@ -156,7 +156,7 @@
   "android_latest": {"type": "real", "model":"flame", "version":"29"},
   "emulator_min": {"type": "virtual", "image":"system-images;android-18;google_apis;x86"},
   "emulator_target": {"type": "virtual", "image":"system-images;android-28;google_apis;x86_64"},
-  "emulator_latest": {"type": "virtual", "image":"system-images;android-29;google_apis;x86_64"},
+  "emulator_latest": {"type": "virtual", "image":"system-images;android-29;google_apis;x86"},
```

Comment:
Why 32-bit?

Reply:
firestore has errors on latest x86_64 devices. They are look on it: b/192351260

Reply:
It looks like Firestore is fixing these errors (or maybe has fixed them already). Assuming that change goes in, we should keep this to 64-bit.

HOWEVER, I actually think it might make sense to check on at least one 32-bit device as part of the expanded matrix. Maybe we add an emulator_32bit?